Fix clang 20 build error: no-switch-default-label

<!-- (At least include the following, feel free to add if you have more content) -->

### RaftKeeper version
v2.2.0
### Description

```
In file included from /data1/home/wujianchao/project/jd/raftkeeper/base/common/getPageSize.cpp:1:
In file included from /data1/home/wujianchao/project/jd/raftkeeper/base/common/getPageSize.h:3:
/data1/home/wujianchao/project/jd/raftkeeper/base/common/types.h:269:5: error: 'switch' missing 'default' label [-Werror,-Wswitch-default]
  269 |     switch (idx)
      |     ^
1 error generated.
```
### How to reproduce

--- a/base/common/CMakeLists.txt
+++ b/base/common/CMakeLists.txt
@@ -29,7 +29,7 @@ else ()
     target_compile_definitions(common PUBLIC WITH_COVERAGE=0)
 endif ()
 
-target_compile_options(common PUBLIC -Wno-implicit-const-int-float-conversion -Wno-reserved-identifier)
+target_compile_options(common PUBLIC -Wno-implicit-const-int-float-conversion -Wno-reserved-identifier -Wno-switch-default)
 
 if (USE_INTERNAL_CCTZ)
     set_source_files_properties(DateLUTImpl.cpp PROPERTIES COMPILE_DEFINITIONS USE_INTERNAL_CCTZ)

